.github/workflows/ci.yml

```diff
@@ -128,8 +127,8 @@ jobs:
                              qtmultimedia5-dev libxml2-dev liburdfdom-dev libtinyxml-dev \
                              liburdfdom-dev liboctave-dev python3-dev valgrind coinor-libipopt-dev \
                              libmatio-dev python3-pytest python3-numpy python3-scipy \
-                             python3-setuptools  libspdlog-dev  libopencv-dev libpcl-dev \
-                             python3-pybind11 nlohmann-json3-dev libassimp-dev
+                             python3-setuptools  libspdlog-dev libopencv-dev libpcl-dev \
+                             python3-pybind11 nlohmann-json3-dev libassimp-dev libqhull-dev
```

Comment:
This package only install the headers and `libqhull.so`, to have `libqhull_r.so` you also need to install `libqhull-r7` in old distro or `libqhull-r8.0` in newer one. However, probably this is already a dependency of pcl, so everything works.